### DISCUSSION TITLE
About the display order of environment

### DISCUSSION THREAD

**Discussion by Toyo-Daichi** [community/community#38645] [Actions] [answered]
Hello.
*This is a GUI request, and I apologize if the category is wrong.

We manage code in separate repositories for the front end and the back end of one application.

Each of them executed CICD by using environment `workflow_dispatch` on Github Actions .
The display of environment in repository is in the order of registration, and there are differences for each repository as shown below.

Since the number of environments will increase in the future, we would like to avoid having different display order for each repository in terms of management. I would be happy that we can change the environment order even after setting.

- Front end Repository
![image](https://user-images.githubusercontent.com/43840739/200751273-c925a129-2132-4659-be29-ba0027cb9e9e.png)

- Back end Repository
![image](https://user-images.githubusercontent.com/43840739/200751358-b0e848c4-55a6-4edf-bea2-dd11336762ae.png)

I would appreciate it if you could consider the next update.

**Comment by ossanna16** [comment, marked as answer]
Hi there @Toyo-Daichi and welcome to our community! Thank you for the feedback!
